docs/API/API-socket-options.md

```diff
@@ -474,7 +474,7 @@ The default flow control window size is 25600 packets. It is approximately:
 - 300 Mbits of payload with an SRT payload size of 1456 bytes.
 
 The minimum number of packets in flight should be (assuming max payload size):
-`FCmin = bps / 8 × RTTsec / (MSS - 44)`,
+`FCmin = bps / 8 × latency_sec / (MSS - 44)`,
```

Comment:
This change does not belong to this PR.
RTTsec is correct here.

Reply:
This was supposed to go in another PR, I must have forgotten to unmark it before commiting

Reply:
Although it should probably be `RTTsec + latency_sec` if we include unacknowledged packets in the RCV buffer.
Still, in a separate PR.